### ISSUE TITLE
TLS 1.2 Support

### ISSUE THREAD

**Issue by wpkramer** [sjdirect/abot#176]
The crawl of a site supporting only TLS 1.2 was failing.  Modifying the constructors of  the PageRequester assigning the default security protocol resolves the problem for us.  This may not be the best home for assigning the SecurityProtocol setting.  Also does not future proof for TLS 1.3.

            System.Net.ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls12 | SecurityProtocolType.Tls11 | SecurityProtocolType.Tls;

**Comment by sjdirect** [comment]
Thanks for reporting this. I agree that Abot may not be the best initializer of this setting. I will leave this issue open so others have a workaround.

**Comment by alex-jitbit** [comment]
Thanks for leaving the issue open @sjdirect !!!

Took me a while to find it though. Adding the error text I was getting: "Could not create SSL/TLS secure channel" (just so Google indexes it and brings more people to the workaround.)

**Comment by sjdirect** [comment]
agreed @alex-jitbit , changed the subject for better phase indexing

**Comment by rarunkumar454** [comment]
Hi, I want to use your DLL. There is any tutorial for your dll please let me know?
